gpii/node_modules/eventLog/src/eventLog.js

```diff
@@ -0,0 +1,396 @@
+/*
+ * Event logging.
```

Comment:
A little more context here would be helpful - since there's been quite a lot of discussion about the scope and purpose behind this module, e.g. that it produces structured logs in JSON that will be streamed into particular kinds of backend servers for particular kinds of analyses etc.